tests/storage/storage.go

```diff
@@ -723,15 +722,6 @@ var _ = SIGDescribe("Storage", func() {
 						hostpath := filepath.Join(testsuite.HostPathBase, pvc)
 						node = libstorage.CreateHostPathPv(pvc, testsuite.GetTestNamespace(nil), hostpath)
 						libstorage.CreateHostPathPVC(pvc, testsuite.GetTestNamespace(nil), "1G")
-						if checks.HasFeature(virtconfig.NonRoot) {
-							nodeSelector = map[string]string{"kubernetes.io/hostname": node}
-							By("changing permissions to qemu")
-							args := []string{fmt.Sprintf(`chown 107 %s`, hostpath)}
-							pod := tests.RenderHostPathPod("tmp-change-owner-job", hostpath, k8sv1.HostPathDirectoryOrCreate, k8sv1.MountPropagationNone, []string{"/bin/bash", "-c"}, args)
-
-							pod.Spec.NodeSelector = nodeSelector
-							tests.RunPodAndExpectCompletion(pod)
-						}
```

Comment:
I raised both storage test changes in the SIG-storage meeting today.
This code was not essential for the test to pass because of changes in KubeVirt/CDI.
@mhenriks do you want me to add additional details to the commit messages?

Reply:
@xpivarc Can't remember why we needed this. Doesn't virt-handler do this for us

Reply:
I believe this was just an initial limitation. See https://github.com/kubevirt/kubevirt/pull/6041 where this was added and https://github.com/kubevirt/kubevirt/pull/6240 where it should have been dropped.